package.json

```diff
@@ -6,7 +6,12 @@
   "license": "MIT",
   "main": "./dist/electron/main.js",
   "scripts": {
-    "release": "node .electron-vue/build.js && electron-builder -mwl",
```

Comment:
According to the Document of `Electron-builder`,  I am not sure weather this config will generate a release draft?

Maybe, we need have a try!

Reply:
I think you have to run `npm run release:all` to generate a draft